modules/infra/control/control_output.c

```diff
@@ -16,20 +16,24 @@
 #include <stdatomic.h>
 #include <stdlib.h>
 
-extern struct rte_ring *control_output_ring;
-static struct event *control_output_evt;
+static struct rte_ring *ctrlout_ring;
```

Comment:
This can be moved to previous patch.

Reply:
indeed ! I messed with the 2 patches. Will fix that.